src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+// Create a new promise (renamed to clickPromise for clarity)
+const clickPromise = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
+
+// Create and append a div to the body
+const div = document.createElement('div');
```

Comment:
It's a good practice to give meaningful names to your variables. Instead of 'div', you could name it 'messageDiv' or 'notificationDiv'.